### ISSUE TITLE
[med] JAWS + Chrome: No Visual label is provided for search text field. 

### ISSUE THREAD

**Issue by SumitDiyora** [Expensify/App#9666]
## Action Performed:
1. Using JAWS+ Chrome, open URL staging.new.expensify.com
2. Navigate to the already created room from the Chat list using the Tab key and press Enter on it.
3. Navigate to the Emoji button located at the bottom of the page, inside the textfield.
4. Navigate to Emoji control using Tab key and press Enter to activate it.
5. A pop-up will appear.
6. Navigate to the Search text field
7. Observe the screen reader's behavior.

## Expected Result:
Search text field must have visible labels so that their purpose should be clear to the user.

## Actual Result:
Search text field doesn't have a visible label. The purpose of the Search text field may not be clear to some users.

## Workaround:
Yes

## Area issue was found in:
Emoji pop-up

## Failed WCAG checkpoints
3.3.2

## User impact:
Some users won't understand the purpose of text fields since the label disappear once user gives the input on the text field.

## Suggested resolution:
Ensure that the Search Text field has a permanently visible label. This can be done using the tag.

Refer to:
https://dequeuniversity.com/rules/axe/4.3/label-title-only

## Platform:
- Web
- Desktop App

**Version Number:*v1.1.79-9*
**Reproducible in staging?:*Yes*
**Reproducible in production?:*Yes*
**Issue reported by:*Sumit*

![no label for textfield](https://user-images.githubusercontent.com/98817464/176910698-426044bc-fe37-4e0e-bee5-75f9c08673d4.jpg)
